### ISSUE TITLE
Normalize capitalization

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#192]
Capitalization style for Discovery is title case. Normalize this for the following pages/components using all caps.
**Search Results**
- [x] Sort by component (currently class=`nypl-results-sorting-controls`)
- [x] Headers in Location tables (and anything else we can control with the toolkit. Some values are being provided by the api and we'll address their capitalization there.)
- [x] Request button in the Locations table

**Bib page**
- [x] "Back to results" link
- [x] All bibliographic fields
- [x] Headers in location table (and anything else we can control with the toolkit. Some values are being provided by the api and we'll address their capitalization there.)
- [x] Request buttons

**Request page**
- [x] "Back to results" link
- [x] Call number field
- [x] Submit request button

**EDD form page**
- [x] "Back to results" link
- [x] Call number field
- [x] Submit request button

**Request confirmation page**
- [ ] "Back to results" link
- [ ] Back to results button
- [ ] Start over button

There may be other components/fields which need normalizing, if so add them to this ticket.

**Comment by wlla** [comment]
I have been tasked with further documenting this punch-list. I will update and reassign this ticket once there are actionable tasks.

**Comment by seanredmond** [comment]
@ricardoom Some of these are button and label treatments.

**Comment by ricardoom** [comment]
@wlla & @seanredmond Basically anywhere we have all caps we'll need to remove `text-transform: uppercase` ?

**Comment by ricardoom** [comment]
This issue will be resolved w/ the next TK update & `npm` publish. I dont have an example for the last set of items, but they should be covered.
